packages/volto/src/components/manage/Sidebar/ObjectBrowserBody.jsx

```diff
@@ -441,40 +466,63 @@ class ObjectBrowserBody extends Component {
           </button>
         </header>
         <Segment secondary className="breadcrumbs" vertical>
-          <Breadcrumb>
-            {this.state.currentFolder !== '/' ? (
-              this.state.currentFolder.split('/').map((item, index, items) => {
-                return (
-                  <React.Fragment key={`divider-${item}-${index}`}>
-                    {index === 0 ? (
-                      <Breadcrumb.Section onClick={() => this.navigateTo('/')}>
-                        <Icon
-                          className="home-icon"
-                          name={homeSVG}
-                          size="18px"
-                        />
-                      </Breadcrumb.Section>
-                    ) : (
-                      <>
-                        <Breadcrumb.Divider key={`divider-${item.url}`} />
-                        <Breadcrumb.Section
-                          onClick={() =>
-                            this.navigateTo(items.slice(0, index + 1).join('/'))
-                          }
-                        >
-                          {item}
-                        </Breadcrumb.Section>
-                      </>
-                    )}
-                  </React.Fragment>
-                );
-              })
-            ) : (
-              <Breadcrumb.Section onClick={() => this.navigateTo('/')}>
-                <Icon className="home-icon" name={homeSVG} size="18px" />
-              </Breadcrumb.Section>
-            )}
-          </Breadcrumb>
+          {this.props.mode === 'image' && (
+            <Icon
+              name={this.state.view === 'list' ? iconsSVG : listSVG}
+              size="24px"
+              className="mode-switch"
+              onClick={this.toggleView}
+            />
+          )}
+          {!this.state.showSearchInput ? (
+            <Breadcrumb>
+              {this.state.currentFolder !== '/' ? (
+                this.state.currentFolder
+                  .split('/')
+                  .map((item, index, items) => {
+                    return (
+                      <React.Fragment key={`divider-${item}-${index}`}>
+                        {index === 0 ? (
+                          <Breadcrumb.Section
+                            onClick={() => this.navigateTo('/')}
+                          >
```

Comment:
I am not sure what this renders but since it can be clicked it should be a button or maybe at least have role="button". I am less experienced with aria roles but it might be enough.